Snakefile

```diff
@@ -0,0 +1,8 @@
+include: "1_fetch.smk"
```

Comment:
So here's where rules associated with multiple phases of the code would be pulled together in to a master file, much like `_targets.R`?

Reply:
Exactly. I believe if you call `snakemake` from the command line without specifying a file, it defaults to `Snakefile`.